Libraries/Components/ToastAndroid/ToastAndroid.android.js

```diff
@@ -22,6 +22,12 @@ var RCTToastAndroid = require('NativeModules').ToastAndroid;
  *
  * There is also a function `showWithGravity` to specify the layout gravity. May be
  * ToastAndroid.TOP, ToastAndroid.BOTTOM, ToastAndroid.CENTER
+ * 
+ * Basic usage:
+ * ```javascript
+ * import { ToastAndroid } from 'react-native';
+ * ToastAndroid.show('A pikachu appeared nearby !', ToastAndroid.SHORT);
```

Comment:
I'd move this example to before the note about `showWithGravity`, also maybe add an example for `showWithGravity` too.